src/utils/util.js

```diff
@@ -0,0 +1,8 @@
+var Util = {};
+
+// NOTE: Some web-sites override String.prototype.trim method
+Util.trim = function (str) {

```

Comment:
1. I think it would be better to rename utility folder to "util" instead of "utils" as it is done on the client
2. We will rename the "util.js" file to a more appropriate.